### PR TITLE
Added support for lvim

### PR DESCRIPTION
**Pull request by LeviticusNelson**
Some who have lvim as their editor couldn't use the key bindings for the vim pane navigation, so I added support for lvim.

### CODE REVIEW

File: vim-tmux-navigator.tmux [hunk 1]
```diff
@@ -3,7 +3,7 @@
 version_pat='s/^tmux[^0-9]*([.0-9]+).*/\1/p'
 
 is_vim="ps -o state= -o comm= -t '#{pane_tty}' \
-    | grep -iqE '^[^TXZ ]+ +(\\S+\\/)?g?(view|n?vim?x?)(diff)?$'"
+    | grep -iqE '^[^TXZ ]+ +(\\S+\\/)?g?(view|l?n?vim?x?)(diff)?$'"
```
Comment (christoomey) [inline vim-tmux-navigator.tmux:6]: Could you update the regex reference in the README as well, @LeviticusNelson ?

**Comment by pupgray** [comment]
This branch works great on my end!

**Comment by LeviticusNelson** [comment]
@christoomey I fixed the documentation and fixed the merge conflicts so it is ready when you are!

**Comment by christoomey** [comment]
Hi @LeviticusNelson, it looks like your editor may have reformatted the files adding some unrelated changes. I went ahead and reimplemented directly in 56603a9a5064e7a4e1506ebe90615cd4a96fd9be, noting you as co-author, and also added `lvim` to the `pattern-check` to make sure this functions as expected. Closing this PR now as the core changes have been merged into master.

**Pull request closed without merging**